src/box/box.cc

```diff
@@ -5963,18 +5963,20 @@ box_broadcast_id(void)
 	assert((size_t)(w - buf) < sizeof(buf));
 }
 
-static void
+void
 box_broadcast_status(void)
 {
 	char buf[1024];
 	char *w = buf;
-	w = mp_encode_map(w, 3);
+	w = mp_encode_map(w, 4);
 	w = mp_encode_str0(w, "is_ro");
 	w = mp_encode_bool(w, box_is_ro());
 	w = mp_encode_str0(w, "is_ro_cfg");
 	w = mp_encode_bool(w, cfg_geti("read_only"));
 	w = mp_encode_str0(w, "status");
 	w = mp_encode_str0(w, box_status());
+	w = mp_encode_str0(w, "schema_version");
```

Comment:
Wouldn't it be better to name the new field `dd_version` so as not to confuse with the schema version we use in IPROTO?

Reply:
I agree that it is better to use different names, but I think that `dd_version` is closer to DD itself rather than its schema/format/layout. I can propose `dd_layout` here.

Reply:
I find name `dd_layout` strange, I suppose, we should be consistent with internal variable naming (`dd_version_id`). Let's use `dd_version` in `box.info`. @Totktonada, what do you think?

Reply:
If it is to follow an existing naming (that doen't not interfere with 'schema version'), I'm OK. Well, I think that naming could be better if we implement it from scratch, but the consistency seems more important for me.

Am I understand right: we're going to add `box.info.dd_version` and `dd_version` field into `box.status` event? If so, I'm OK.

Reply:
> we're going to add box.info.dd_version

Do we really need that? User can get it from `_schema` space. We already duplicate that, but in `box.status` it's for convenience of upgrade process. Why do we want to add it to the `box.info`?

Reply:
Reading a specific value from a system space is not something that we should normally suggest to a user. A system space layout/content is a kind of public API due to historical reasons, at least for some spaces. But if something is going to be widely used/too visible, I would add it into some 'undoubtedly public' API.

Also, this idea appears in the version module RFC discussion and it seems meaningful for me.

I don't insist, but I think that it is a good idea.